src/main/java/Domain/Game/Season.java

```diff
@@ -1,4 +1,50 @@
 package Domain.Game;
 
+import java.time.Year;
+import java.util.ArrayList;
+import java.util.List;
+
 public class Season {
+
+    private League league;
+    private String years; //name
+    List<Team> teams;
+
+    public Season(League league, String years) {
+        this.league = league;
+        this.teams = new ArrayList<>();
+        this.years = years;
+    }
+
+    public Year getYear() {
+        int[] yearsInt = splitYearsInt(years);
+        return Year.parse(""+yearsInt[0]+yearsInt[2]);
+    }
+
+    public List<Team> getTeams() {
+        return teams;
+    }
+
+    public boolean addTeam(Team team){
+        if(!teams.contains(team)){
+            teams.add(team);
+            return true;
+        }
+        return false;
+    }
+
+    private static String[] splitYearsString(String years){
```

Comment:
Is this function used?

Reply:
Ask Yiftach @yiftachsa :), I kidding, yes it has been used